internal/organizations/delete.go

```diff
@@ -58,19 +59,18 @@ func (c *Client) Delete() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			if !force {
-				prompt := promptui.Prompt{
-					Label:     fmt.Sprintf("Are you sure you want to delete organization %s: ", organizationID),
-					IsConfirm: true,
-				}
 
-				_, err := prompt.Run()
-				if err != nil {
+			if !force {
+				fmt.Printf("Are you sure you want to delete device %s (y/N): ", organizationID)
+				userInput := prompt.Input(">", func(d prompt.Document) []prompt.Suggest {
```

Comment:
In this PR, most of the prompts are not using suggestions. If we choose not to use the suggestion feature, we are better off using a native [fmt.Sscanln](https://pkg.go.dev/fmt#Sscanln) and avoiding any dependency.

On the other hand, prompt suggestions ([shown here](https://github.com/c-bata/go-prompt#go-prompt) and implemented for [port convert below](https://github.com/equinix/metal-cli/pull/399/files#diff-04080aab6a2c5bc5b599c51a90e52ef2ba3a6244b39a11cca412f92423bdd9bfR70)) could offer a better interface for our `metal init` steps and could be used when a user is present to prompt (with suggestions) when a required value has been omitted (we can detect a terminal or stdin).

Reply:
I'd want to avoid introducing a fancy library to replace it with another soon after. Going from our current prompt library to fmt.Scanln would avoid that with no user-facing change in experience.